polyfill/lib/ecmascript.mjs

```diff
@@ -1830,7 +1830,7 @@ export const ES = ObjectAssign({}, ES2022, {
     }
     return offsetNs;
   },
-  BuiltinTimeZoneGetOffsetStringFor: (timeZone, instant) => {
+  GetOffsetStringFor: (timeZone, instant) => {
```

Comment:
Oh, I thought it was "built-in (TimeZone#getOffsetStringFor)", not "(built-in TimeZone) getOffsetStringFor". Either way is fine, though.